inductiva/structures/scenarios/plate_linear_elastic/geometry_utils.py

```diff
@@ -0,0 +1,413 @@
+"""Utils to create the geometric case."""
+
+from typing import List, Optional
+
+import gmsh
+import json
+import math
+
+
+class RectangularPlate:
+    """Rectangular plate.
+
+    Attributes:
+        plate_length: A float representing the plate length.
+        plate_width: A float representing the plate width.
```

Comment:
These are properties of the plate, so no need for the `plate_` prefix. Use `width` and `height` only.